librz/bin/p/bin_mdmp.c

```diff
@@ -57,7 +57,7 @@ static RzBinInfo *mdmp_info(RzBinFile *bf) {
 
 	obj = (MiniDmpObj *)bf->o->bin_obj;
 
-	ret->big_endian = obj->endian;
+	ret->big_endian = false;
```

Comment:
why?

Reply:
the obj->endian was never initialized. the only code that was initializing it was when `RZ_NEW0(MiniDmpObj)` was called for

Reply:
but then it never gets assigned in the whole code.